apps/qubit/modules/settings/actions/diacriticsAction.class.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class SettingsDiacriticsAction extends SettingsEditAction
+{
+    // Arrays not allowed in class constants
+    public static $NAMES = [
+        'diacritics',
+    ];
+
+    public function earlyExecute()
+    {
+        parent::earlyExecute();
+
+        $this->updateMessage = $this->i18n->__('Diacritics settings saved.');
+
+        $this->settingDefaults = [
+            'diacritics' => 0,
+        ];
+    }
+
+    public function execute($request)
+    {
+        parent::execute($request);
+    }
+
+    public function uploadDiacritics($request)
+    {
+        $file = $request->getFiles('mappings');
+        $diacriticsMappingPath =
+            sfConfig::get('sf_plugins_dir') . DIRECTORY_SEPARATOR .
+            'arElasticSearchPlugin' . DIRECTORY_SEPARATOR .
+            'config' . DIRECTORY_SEPARATOR .
+            'diacritics_mapping.yml';
```

Comment:
We usually put config files overwriting the defaults in the main config folder, as it's easier to keep track of those configuration changes. And the way the file is read below, the main config folder will take precedence already:

https://github.com/artefactual/atom/pull/1636/files#diff-5f879544d637140233df54dbb8f025b8a59f4b150fd5bdb38d89e9493d12f0bfR142-R145